src/app/shared/utils/consts.ts

```diff
@@ -171,6 +174,16 @@ export const Consts = {
                 value: 'ibm'
             }
         ],
+        resources:{
+            volumes : ['vmax', 'unity', 'vnx', 'oceanstor', '3par', 'vsp', 'storwize_svc', 'fake_driver'],
```

Comment:
resource list need to be based on the current backend support.

current backend support is 
resources:{
            volumes : ['vmax', 'unity', 'vnx', 'oceanstor', '3par', 'vsp', 'storwize_svc', 'fake_driver'],
            pools : ['vmax', 'unity', 'vnx', 'oceanstor', '3par', 'vsp', 'storwize_svc', 'fake_driver'],
            controllers : ['oceanstor', 'fake_driver']
            ports : ['oceanstor', 'fake_driver']
            disks : ['oceanstor', 'fake_driver'],
            qtrees : ['oceanstor', 'fake_driver'],
            filesystems : ['oceanstor', 'fake_driver'],
            shares: ['oceanstor', 'fake_driver']
        },

Reply:
Thanks @NajmudheenCT I have updated the PR with this. 
I removed `fake_driver` as it is only needed for development.